### PR TITLE
01334 d increase reconnect tests tps

### LINKED ISSUE #1334: Increase TPS on existing reconnect tests
**Summary of the defect**
Currently all existing reconnect tests does a TPS of 1.5K . Increase the TPS in the existing reconnect tests to 5K

### PR DESCRIPTION
**Pull request by ljianghedera**
**Related issue(s)**:
Closes #1334 #1361

**Summary of the change**:

1. Added/converted high TPS tests for reconnect test;
2. Also fixed the fee initialization corruption issue;
3. Merged reconnect time window too;

Successful slack links: https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621299271001300
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621296973000900
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621299368001800
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621364710007000
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621456973024600

etc.

Summary links: https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621299421002200
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621299316001700
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621306473003100
https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1621459515026400

All high TPS reconnect tests are fine except the MixedOps-NIReconnect-6k-28m. An issue: https://github.com/hashgraph/hedera-services/issues/1431 is opened to track it.

**External impacts**:
None.

**Applicable documentation**
- [ ] Javadoc
- [ ] HAPI protobuf comments
- [ ] README

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AdjustFeeScheduleSuite.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
+package com.hedera.services.bdd.suites.perf;
+
```
Comment (qnswirlds) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AdjustFeeScheduleSuite.java:2]: missing license header

Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AdjustFeeScheduleSuite.java:2]: Added.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AdjustFeeScheduleSuite.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.reduceFeeFor;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.sleepFor;
+import static com.hederahashgraph.api.proto.java.HederaFunctionality.ConsensusSubmitMessage;
+import static com.hederahashgraph.api.proto.java.HederaFunctionality.CryptoTransfer;
+
+
+/**
+ * This is a suite for any reconnect tests with multiple clients that needs to adjust fee schedules
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AdjustFeeScheduleSuite.java:40]: Pease add comment, why we are adjusting fee schedules for reconnect tests

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/topic/SubmitMessageLoadTest.java [hunk 3]
```diff
@@ -148,7 +146,6 @@ private static HapiApiSpec runSubmitMessages() {
 								.payingWith(GENESIS)
 								.overridingProps(Map.of("hapi.throttling.buckets.fastOpBucket.capacity", "4000",
 										"hapi.throttling.ops.consensusSubmitMessage.capacityRequired", "1.0")),
-						reduceFeeFor(ConsensusSubmitMessage, 2L, 3L, 3L),
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/topic/SubmitMessageLoadTest.java:151]: Why is this line removed? Will this not cause any issues in perf tests ?

File: test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/SubmitMessagesForReconnect.java [hunk 4]
```diff
@@ -54,15 +54,20 @@ public static void main(String... args) {
 		);
 	}
 
+	private static HapiSpecOperation submitToTestTopic(PerfTestLoadSettings settings) {
+		String topicToSubmit = String.format("0.0.%d",	settings.getTestTopicId());
+		return submitMessageTo(topicToSubmit)
+				.message(randomUtf8Bytes(100))
+				.noLogging()
+				.hasRetryPrecheckFrom(BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED)
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/SubmitMessagesForReconnect.java:62]: DUPLICATE_TRANSACTION should be removed for retries

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/token/TokenTransfersLoadProvider.java [hunk 5]
```diff
@@ -179,7 +177,6 @@ private HapiApiSpec runTokenTransfers() {
 						spec.tryReinitializingFees();
 					}));
 				}
-				initializers.add(reduceFeeFor(CryptoTransfer, 2L, 3L, 3L));
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/token/TokenTransfersLoadProvider.java:182]: Why is this line removed? Will this not cause any issues in perf tests ?

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/crypto/CryptoTransferLoadTest.java [hunk 6]
```diff
@@ -117,7 +119,6 @@ protected HapiApiSpec runCryptoTransfers() {
 						fileUpdate(APP_PROPERTIES)
 								.payingWith(GENESIS)
 								.overridingProps(Map.of("hapi.throttling.buckets.fastOpBucket.capacity", "1300000.0")),
-						reduceFeeFor(CryptoTransfer, 2L, 3L, 3L),
```
Comment (anighanta) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/perf/crypto/CryptoTransferLoadTest.java:120]: Removed from here as well !

**Review by anighanta — COMMENTED** [review]
see inline comments

File: .circleci/config.yml [hunk 7]
```diff
@@ -1551,6 +1551,58 @@ workflows:
...
+            branches:
+              only:
+#                - master
+                - X-01334-D-increase-reconnect-tests-TPS
+    jobs:
+      - build-platform-and-services
+      - jrs-regression:
+          context: Slack
+          regression_path: /swirlds-platform/regression
+          result_path: results/15N_4C/Reconnect
```
[the remark this answers is not in the document]
Comment (ljianghedera) [reply .circleci/config.yml:1592]: Done.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AdjustFeeScheduleSuite.java (hunk 2 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/AdjustFeeScheduleSuite.java:40]: Changed the comments. This is not only for reconnect tests. It's for any high volume requests that may run into the situation of INSUFFICIENT_PAYER_BALANCE.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/token/TokenTransfersLoadProvider.java (hunk 5 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/token/TokenTransfersLoadProvider.java:182]: The `reduceFeeFor(...)` operation may cause the random fee initialization corruption issue if running in middle of tests with multiple clients. That's why I add the `AdjustFeeScheduleSuite` to do the fee reduction at the beginning to avoid it.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/crypto/CryptoTransferLoadTest.java (hunk 6 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/crypto/CryptoTransferLoadTest.java:120]: See below my comment.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/perf/topic/SubmitMessageLoadTest.java (hunk 3 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/perf/topic/SubmitMessageLoadTest.java:151]: See the comment above.

File: test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/SubmitMessagesForReconnect.java (hunk 4 above)
Comment (ljianghedera) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/SubmitMessagesForReconnect.java:62]: Done

File: .circleci/config.yml [hunk 8]
```diff
@@ -1551,6 +1551,58 @@ workflows:
...
+
+  GCP-Daily-Services-Global-3NReconnect-15N-4C:
+    triggers:
+      - schedule:
+          cron: "30 3,11 * * *"
+          filters:
+            branches:
+              only:
+#                - master
+                - X-01334-D-increase-reconnect-tests-TPS
```
[the remark this answers is not in the document]
Comment (ljianghedera) [reply .circleci/config.yml]: This is a transitive state. Already fixed.

File: .circleci/config.yml [hunk 9]
```diff
@@ -1714,45 +1767,23 @@ workflows:
...
     triggers:
       - schedule:
-          cron: "15 8 * * *"
+          cron: "30 3,11 * * *"
           filters:
             branches:
               only:
-                - master
+#                - master
+                - X-01334-D-increase-reconnect-tests-TPS
```
[the remark this answers is not in the document]
Comment (ljianghedera) [reply .circleci/config.yml]: Changed.

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Review by abhishek-hedera — APPROVED** [review]
